src/GeoPing.Api/Models/Entities/UserPoint.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace GeoPing.Api.Models.Entities
+{
+    public class UserPoint
```

Comment:
it should has explicit Id  in order to Entity Framework could track it!

Reply:
done